apps/codeforafrica/package.json

```diff
@@ -38,12 +38,14 @@
     "@emotion/styled": "^11.8.1",
     "@mui/material": "^5.6.4",
     "@mui/utils": "^5.6.1",
+    "@svgr/webpack": "^6.2.1",
     "next": "^12.1.6",
     "next-transpile-modules": "^9.0.0",
     "prop-types": "^15.8.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-share": "^4.4.0"
+    "react-share": "^4.4.0",
+    "webpack": "^5.72.0"
```

Comment:
`@svgr/webpack` is already included in `devDependencies`... and not sure if we need to include `webpack` since Next.js already includes it. **If we do**, `webpack` should also be in `devDependencies`.